From 4623436238ceed9d4d4f73ea3e9a2229d85103cb Mon Sep 17 00:00:00 2001
From: recon <recon@local>
Date: Thu, 8 Oct 2026 08:23:19 +0300
Subject: [PATCH 1/3] build plumbing: restrict top Kbuild to
 drivers/gpu/mediatek/ (stock built the whole tree; only object selection
 changes)

---
 Kbuild | 49 -------------------------------------------------
 1 file changed, 49 deletions(-)

diff --git a/Kbuild b/Kbuild
index 8f7faae..d8b6838 100644
--- a/Kbuild
+++ b/Kbuild
@@ -8,104 +8,55 @@ subdir-ccflags-y += -Werror \
 		-I$(srctree)/$(src)/include \
 		-I$(srctree)/$(src)/include/uapi \
 
-obj-y += drivers/memory/
 
-obj-y += drivers/iio/adc/
 
-obj-y += drivers/mfd/
 
-obj-y += drivers/nvmem/
 
-obj-y += drivers/dma/mediatek/
 
-obj-y += drivers/ufs/
 
-obj-y += drivers/char/
 
-obj-y += drivers/clk/mediatek/
 
-obj-y += drivers/clocksource/
 
-obj-y += drivers/cpufreq/
 
-obj-y += drivers/soc/mediatek/
 
-obj-y += drivers/watchdog/
 
-obj-y += drivers/dma-buf/heaps/
 
-obj-y += drivers/regulator/
 
-obj-y += drivers/leds/
 
-obj-y += drivers/pinctrl/mediatek/
 
-obj-y += drivers/power/supply/
 
-obj-y += drivers/rtc/
 
-obj-y += drivers/remoteproc/
 
-obj-y += drivers/rpmsg/
 
-obj-y += drivers/input/keyboard/
 
-obj-y += drivers/input/fingerprint/goodix/
 
-obj-y += drivers/phy/mediatek/
 
-obj-y += drivers/thermal/mediatek/
 
-obj-y += drivers/thermal/nothing/
 
-obj-y += drivers/spmi/
 
-obj-y += drivers/tty/serial/8250/
 
-obj-y += drivers/reset/
 
-obj-y += drivers/mailbox/
 
-obj-y += drivers/interconnect/
 
-obj-y += drivers/i2c/busses/
 
-obj-y += drivers/pwm/
 
-obj-y += drivers/spi/
 
-obj-y += drivers/iommu/
 
-obj-y += drivers/mmc/host/
 
-obj-y += drivers/tee/
 
-obj-y += drivers/gpu/drm/mediatek/
 
-obj-y += drivers/input/touchscreen/
 
-obj-y += drivers/gpu/drm/panel/
 
 obj-y += drivers/gpu/mediatek/
 
-obj-y += drivers/media/platform/
 
-obj-y += drivers/usb/
 
-obj-y += drivers/devfreq/
 
-obj-y += drivers/misc/mediatek/
 
-obj-y += drivers/misc/haptic/
 
-obj-y += sound/soc/codecs/
 
-obj-y += sound/soc/mediatek/
 
-obj-y += drivers/pci/controller/
 
-obj-y += drivers/chino-e/
 
-obj-y += drivers/nothing_stability/
 
 endif
-- 
2.56.0

